src/CMakeLists.txt

```diff
@@ -557,6 +557,11 @@ if (TARGET ch_contrib::accel-config)
     dbms_target_link_libraries(PUBLIC ch_contrib::accel-config)
 endif ()
 
+if (TARGET ch_contrib::qatzstd_plugin)
```

Comment:
Cosmetic: While at it, please indent l. 553.

Reply:
Done